### PR TITLE
Add CS workflow

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

 - [x] Adds code style checks since Travis CI is out of use.
 - [x] Uses all necessary caching to ensure fastest builds.

GitHub actions might [need to be enabled](https://github.com/beaulebens/keyring/settings/actions) for this repository, but this is [as hard as checking a checkbox](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository#managing-github-actions-permissions-for-your-repository).

If all goes well, I'll try my luck with tests.

### CODE REVIEW

File: .github/workflows/cs.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    name: PHPCS
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          tools: composer:v1
```
Comment (sanmai) [inline .github/workflows/cs.yml:29]:
```suggestion
          tools: composer:v2
```

**Comment by sanmai** [comment]
I wonder if there was a reason this PR became undesired. It should be possible to squash-merge it.

**Comment by beaulebens** [comment]
Hrm, I think this got caught up in me fixing the commit history of this entire repo :-/

I wasn't intentionally doing anything to this one, and am honestly not actually clear on what has happened.

**Comment by sanmai** [comment]
No problem at all, thanks for following up. Shall I rebase this for you, or shall we squash-merge as is?

**Comment by beaulebens** [comment]
If you could rebase, that'd be great, then I can take a proper look at it and see about getting it merged. Thanks!

**Comment by sanmai** [comment]
Rebased. GitHub is quite slow to catch up.

**Pull request closed without merging**